lib/livebook/system_resources.ex

```diff
@@ -46,13 +46,16 @@ defmodule Livebook.SystemResources do
 
   @impl true
   def handle_cast(:update, state) do
-    measure()
+    memory = measure()
+    Phoenix.PubSub.broadcast(Livebook.PubSub, "system_resources", {:memory_update, memory})
```

Comment:
```suggestion
    Phoenix.PubSub.local_broadcast(Livebook.PubSub, "system_resources", {:memory_update, memory})
```

Or whatever the API is :D